templates/config-map-aws-auth.yaml.tpl

```diff
@@ -6,8 +6,8 @@ metadata:
 data:
   mapRoles: |
 ${worker_role_arn}
-${map_roles}
+    ${indent(4, map_roles)}
   mapUsers: |
-${map_users}
+    ${indent(4, map_users)}
   mapAccounts: |
-${map_accounts}
+    ${indent(4, map_accounts)}
```

Comment:
If we indent these by 2 more, I think it will be nicer to read and also will show less diff when your PR is applied. See how the indention here is decreasing by 2:

```
          -     - rolearn: arn:aws:iam::66666666666:role/role1
          -       username: role1
          -       groups:
          -         - system:masters
          -
          +     - "groups":
          +       - "system:masters"
          +       "rolearn": "arn:aws:iam::66666666666:role/role1"
          +       "username": "role1"
```